tests/e2e_test/test_bench.py

```diff
@@ -126,7 +105,7 @@ def run_bench_command(command):
         for future in as_completed(future_to_command):
             try:
                 process = future.result()
-                process.wait(timeout=60*30)
+                process.wait(timeout=60*BENCH_TEST_TIMEOUT_MINS)
 
                 if process.returncode != 0:
                     dump_error_log = True
```

Comment:
call backup_instance_log() in line 111.  as assert process.returncode == 0 in line 113 will exit the test.
The same applies to several other e2e tests.

Reply:
done